boards/arm/qemu_cortex_m0/nrf_timer_timer.c

```diff
@@ -162,18 +162,8 @@ void timer0_nrf_isr(void *arg)
 
 int z_clock_driver_init(struct device *device)
 {
-	struct device *clock;
-
 	ARG_UNUSED(device);
 
-	clock = device_get_binding(DT_LABEL(DT_INST(0, nordic_nrf_clock)));
-	if (!clock) {
-		return -1;
-	}
-
-	clock_control_on(clock, CLOCK_CONTROL_NRF_SUBSYS_HF);
-
-	/* FIXME switch to 1 MHz once this is fixed in QEMU */
```

Comment:
You should leave the FIXME line, this is about an issue in QEMU

Reply:
comment is back